src/components/Loading/Loading.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
```

Comment:
You could make a union type from the animations, and use that as the prop type :)